tests/test_login.py

```diff
@@ -12,8 +12,8 @@ class TestLogin(object):
 
     _fxa_logged_in_indicator_locator = (By.ID, 'loggedin')
 
-    def test_user_can_sign_in(self, mozwebqa, dev_account):
-        fxa = WebDriverFxA(mozwebqa)
+    def test_user_can_sign_in(self, base_url, selenium, dev_account, click_login):
+        fxa = WebDriverFxA(base_url, selenium)
         fxa.sign_in(dev_account.email, dev_account.password)
-        WebDriverWait(mozwebqa.selenium, mozwebqa.timeout).until(
+        WebDriverWait(selenium, 20).until(
```

Comment:
Why the higher timeout here? As we have several hard-coded timeouts, I'm wondering if we should add a `timeout` fixture so we can set this in just one location.